packages/nextjs-kit/tailwind.config.js

```diff
@@ -1,6 +1,12 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
 	content: ['./src/components/**/*.tsx'],
+	safelist: [
+		{
+			pattern: /grid-cols-([0-9])/,
```

Comment:
I think we need the prefix here
```suggestion
			pattern: /ps-grid-cols-([0-9])/,
```

Reply:
Sorry I forgot to add a comment, I tested without the prefix and it seems that it is not necessary.